docs/csharp/language-reference/builtin-types/built-in-types.md

```diff
@@ -33,6 +33,7 @@ The following table lists the C# built-in [reference](../keywords/reference-type
 |--------------|-------------------------|
 |[`object`](reference-types.md#the-object-type)|<xref:System.Object?displayProperty=nameWithType>|
 |[`string`](reference-types.md#the-string-type)|<xref:System.String?displayProperty=nameWithType>|
+|[`dynamic`](reference-types.md#the-dynamic-type)||
```

Comment:
Should there be any text in the empty cell of the ".NET type" column?

Reply:
The dynamic type doesn't map to a different .NET type. I'd put `<xref:System.Object>` as well.

Reply:
Thank you @BillWagner, I've updated the table.

Reply:
@BillWagner ping